packages/fixed-bridge/src/BridgeSide.ts

```diff
@@ -54,8 +54,8 @@ export class BridgeSide {
   *** make its way to the neighbor root of the linked anchor on chain X.
   *** @param linkedAnchorInstance: the anchor instance on the opposite chain
   ***/
-  public async createAnchorUpdateProposalData(linkedAnchorInstance: Anchor) {
-    const proposalData = await linkedAnchorInstance.getProposalData();
+  public async createAnchorUpdateProposalData(linkedAnchorInstance: Anchor, thisAnchorInstance: Anchor) {
+    const proposalData = await linkedAnchorInstance.getProposalData(thisAnchorInstance);
```

Comment:
Lets also just call the get resourceID method and pass that result to the anchor whose state we are fetching.